nicolegooden/indecision-maker
# Testing ng

#### What's this PR do?  

+ Allows the user to automatically see the top of each question set within `Form`, rather than clicking the next button and needing to scroll up to see the first question
+ Removes initial test for App 
+ Adds unit, integration, and sad path testing for `Form`

#### Where should the reviewer start? 

Most changes have occurred in `Form.test.js` so start there to see the 4 tests I've created.  As of now, they are all passing.

#### How should this be manually tested?  

Run `npm test` to verify that the tests from `Form.test.js` are passing as expected.  Also, view the form component in the browser and ensure that you are taken to the top of the window when you click the next button.

#### Any background context you want to provide?  

Please let me know if my `Form` tests need to be more thorough in any way.

#### What are the relevant tickets?  

Closes #28  
Closes #67
